src/v2/Apps/Order/routes.tsx

```diff
@@ -6,17 +6,18 @@ import { Redirect, RedirectException, RouteConfig } from "found"
 import * as React from "react"
 import { graphql } from "react-relay"
 
+import { RespondFragmentContainer as RespondRoute } from "./Routes/Respond"
+import { OfferFragmentContainer as OfferRoute } from "./Routes/Offer"
+import { ShippingFragmentContainer as ShippingRoute } from "./Routes/Shipping"
+import { PaymentFragmentContainer as PaymentRoute } from "./Routes/Payment"
+import { NewPaymentFragmentContainer as NewPaymentRoute } from "./Routes/NewPayment"
+import { CounterFragmentContainer as CounterRoute } from "./Routes/Counter"
+import { ReviewFragmentContainer as ReviewRoute } from "./Routes/Review"
+import { AcceptFragmentContainer as AcceptRoute } from "./Routes/Accept"
+import { RejectFragmentContainer as DeclineRoute } from "./Routes/Reject"
+import { StatusFragmentContainer as StatusRoute } from "./Routes/Status"
+
 const OrderApp = loadable(() => import("./OrderApp"))
-const RespondRoute = loadable(() => import("./Routes/Respond"))
-const OfferRoute = loadable(() => import("./Routes/Offer"))
-const ShippingRoute = loadable(() => import("./Routes/Shipping"))
-const PaymentRoute = loadable(() => import("./Routes/Payment"))
-const NewPaymentRoute = loadable(() => import("./Routes/NewPayment"))
-const CounterRoute = loadable(() => import("./Routes/Counter"))
-const ReviewRoute = loadable(() => import("./Routes/Review"))
-const AcceptRoute = loadable(() => import("./Routes/Accept"))
-const DeclineRoute = loadable(() => import("./Routes/Reject"))
-const StatusRoute = loadable(() => import("./Routes/Status"))
```

Comment:
There's diminishing returns by bundle splitting everything. It's fine to split the top most route and then let it load its children, especially if children are really tiny bits of UI as is the case here.

Reply:
Yup! Makes sense.